metricflow-semantics/tests_metricflow_semantics/snapshots/test_suggestions.py/str/test_suggestions_for_metric__result_0.txt

```diff
@@ -6,7 +6,7 @@ Error #1:
     The given input does not exactly match any known metrics.
 
     Suggestions:
-      ['bookings', 'booking_fees', 'booking_value', 'instant_bookings', 'booking_payments', 'max_booking_value']
+      ['bookings', 'booking_fees', 'booking_value', 'booking_payments', 'instant_bookings', 'booking_value_p99']
```

Comment:
So max_booking_value was tied with booking_value_p99 here and the sort pushed the latter into 6th place?

Reply:
Correct!

Reply:
Well it's actually like a 7-way tie between a bunch of metrics.